src/static/js/onlineforms.js

```diff
@@ -289,6 +290,14 @@ $(function() {
             tableform.buttons_bind(this.buttons);
             tableform.table_bind(this.table, this.buttons);
             this.load_person_flags();
+            $("#filechooser").change(function() {
+                let importfile = $("#filechooser").val();
+                if ( !importfile.toLowerCase().endsWith(".html") && !importfile.toLowerCase().endsWith(".json") ) {
+                    $("#dialog-import .ui-state-error").show();
```

Comment:
If you gave it an id above, you can use $("#typeerror").show(); for legibility and it means if we stop using jquery UI in the future, we aren't dependent on JQUI specific classes